__tests__/cli.test.ts

```diff
@@ -1,10 +1,12 @@
-import { execSync } from "child_process";
+import { execSync, spawnSync } from "child_process";
 
 describe("cli", () => {
   it("should run when no files are found", () => {
-    const result = execSync("npm run typed-scss-modules src").toString();
+    const result = spawnSync("npm run typed-scss-modules src", {
+      shell: true,
+    });
 
-    expect(result).toContain("No files found.");
+    expect(result.stderr.toString()).toContain("No files found.");
```

Comment:
why these changes?

Reply:
@skovy this was needed because of the changes to use `console.warn` where appropriate. NodeJS sends `console.warn` messages to `stderr` (appropriately) -- additionally `execSync` doesn't expose messages in `stderr` of the downstream process. `spawnSync` however does let you inspect `stdout` and `stderr`

Reply:
Got it, that makes sense. While here and making this change, can we make them all `spawnSync` for consistency? and it sounds like it supports more functionality which we need here so others might need this in the future as well